docs/cops_chefcorrectness.md

```diff
@@ -169,7 +169,7 @@ Enabled by default | Supports autocorrection
 --- | ---
 Enabled | Yes
 
-Libraries should be injected into the Chef::DSL::Recipe or Chef::DSL::Resource classes and not Recipe/Resource/Provider classes directly.
```

Comment:
@ncerny if you end up in Cookstyle land again these are actually auto generated from the source so you don't need to update them.